sidecar-advanced-custom/Notes_PostmanEnvironment.json

```diff
@@ -0,0 +1,364 @@
+{
+  "id": "4803c510-c7b6-f0e6-33ae-f2b462e4511e",
+  "name": "uncon2017",
+  "values": [
+    {
+      "enabled": true,
+      "key": "url",
+      "value": "http://localhost/SugarEnt-Full-7.9.1.0",
+      "type": "text"
+    },
+    {
+      "enabled": true,
+      "key": "rest_endpoint",
+      "value": "/rest/v10",
+      "type": "text"
+    },
+    {
+      "enabled": true,
+      "key": "Note_Path",
```

Comment:
This whole file will be removed from version control, only here for demonstration purposes while we're actively working on it! 

This is basically a copy of the environment settings partners will need to setup on Postman for any tutorial, the important and different piece required by our customization is the `Notes_Path` which they'll need to setup on their environment settings, which needs to match they're local path to the images folder.

Reply:
```
newman run Notes_PostmanCollection.json --environment Notes_PostmanEnvironment.json